CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog Short Message Service
+
+# v0.1.0 - Initial version
```

Comment:
It's your very first release. Release will be number sequentially (r0.1, r0.2, ...)

```suggestion
# r0.1
```